onecgiar-pr-client/src/app/pages/type-one-report/pages/tor-key-result-story/tor-key-result-story.component.ts

```diff
@@ -1,3 +1,6 @@
+/* eslint-disable camelcase */
+/* eslint-disable no-console */
```

Comment:
@yecksin if you are removing line 26, please remove this too

Reply:
Understood!